purchase_work_acceptance/models/work_acceptance.py

```diff
@@ -120,6 +120,11 @@ def create(self, vals):
     @api.multi
     def button_accept(self, force=False):
         self._unlink_zero_quantity()
+        po_lines = self.purchase_id.order_line
+        for po_line in po_lines:
+            if po_line.product_id.type == 'service':
+                po_line.qty_received = self.wa_line_ids.filtered(
```

Comment:
Type not in product consu, just for following the standard

Reply:
Edited : https://github.com/OCA/purchase-workflow/pull/774/commits/d48b95022bd9ec1b1bfd9832102f755e5cc709d8